rc/core/c-family.kak

```diff
@@ -196,6 +196,21 @@ addhl -group /cpp/code regex %{\b-?(0x[0-9a-fA-F]+|\d+)[fdiu]?|'((\\.)?|[^'\\])'
     "
 }
 
+# c and c++ compiler macros
```

Comment:
There are no standard pre-processing macro anyway, I thought I would highlight those provided by `gcc` (most of those are supported by `clang` as well), since they are fairly common.

I could trim this list down to the bare minimum (`__FILE__`, `__LINE__` and the likes) if needed, I didn't think it wuold be an issue since I'm using the `function` face and not -say- the `keywords` one.